### DISCUSSION TITLE
Github Education unable to verify your academic affiliation

### DISCUSSION THREAD

**Discussion by BarbDMC** [community/community#138490] [GitHub Education]
### Hi everyone,

I'm trying to renew my GitHub education. However, I keep receiving emails rejecting my application.

![image](https://github.com/user-attachments/assets/0c99b658-ad08-4683-b9a8-e0e2cd2a79f2)

![image](https://github.com/user-attachments/assets/e91086ae-7b16-4550-9b13-1c7a5b74f254)

- I'm using these documents to verify my academic affiliation:

![image](https://github.com/user-attachments/assets/72107ca8-12fb-48fe-b50d-72cb9a26c8d6)
![image](https://github.com/user-attachments/assets/baf00fd8-b9fb-4ab0-9473-8ea1c7824399)

- I've already enabled two-factor authentication, and my last name is the same as my GitHub billing information.

Is there anything else I have to fix?

Thank you very much beforehand, for any help.

**Comment by alx-mp4** [comment]
Hi @BarbDMC, I recommend opening a ticket with GitHub support for this issue. I had applied for the GitHub Education Program myself, but my application was declined. If your photos are blurry, contain a foreign language, or are missing required information, the application is likely to be rejected. Make sure your photos meet all the requirements, and if they’re not in English, include a translation before resubmitting.

If this has helped you, I would appreciate if my answer is marked as an answer. Thank you!
